guides/common/modules/proc_invalidating-jwts-of-all-users.adoc

```diff
@@ -0,0 +1,36 @@
+[id="invalidating-jwts-of-all-users"]
+= Invalidating JWTs of all users
+
+You can invalidate all registration JSON Web Tokens of all users at once.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-invalidating-jwts-of-all-users[].
+To use the API, see the xref:api-invalidating-jwts-of-all-users[].
+
+.Prerequisites
+* Your {Project} user has a role that grants the `view_users` and `edit_users` permissions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
+. Click *Invalidate JWTs for all users*.
+. In the pop-up window, click *Confirm*.
+
+.Verification
+* The {ProjectWebUI} displays the following message: `Successfully invalidated registration tokens for all users.`
+
+[id="cli-invalidating-jwts-of-all-users"]
+.CLI procedure
+* Invalidate all registration tokens of all users by running Hammer:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+$ hammer
+----
+
+[id="api-invalidating-jwts-of-all-users"]
+.API procedure
+* Invalidate all registration tokens of all users by calling the following API endpoint:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+$ curl
+----
```

Comment:
Is there an Ansible module too?
```suggestion
----

.Ansible procedure
```

Reply:
I don't think there will be. @girijaasoni Can you please confirm?